controllers/create_pgconfigmap.go

```diff
@@ -20,7 +20,7 @@ func (r *SearchReconciler) PostgresConfigmap(instance *searchv1alpha1.Search) *c
 			Namespace: ns,
 		},
 	}
-	work_mem := r.GetDBConfig(context.TODO(), instance, "WORK_MEM")
+	work_mem := r.GetDBConfigFromSearchCR(context.TODO(), instance, "WORK_MEM")
```

Comment:
We use this in line 46 to create work_memquery.
work_memquery := "psql -d search -U searchuser -c \"ALTER ROLE searchuser set work_mem='" + work_mem + "'\""
Do we need to do this for other variables that the user adds?

Reply:
Discussed in scrum and agreed that this will be a separate user story.